code/game/gamemodes/cult/talisman.dm

```diff
@@ -270,8 +270,13 @@
 	var/mob/living/carbon/human/H = user
 	user.visible_message("<span class='warning'>Otherworldly armor suddenly appears on [user]!</span>", \
 						 "<span class='cultitalic'>You speak the words of the talisman, arming yourself!</span>")
-	H.equip_or_collect(new /obj/item/clothing/suit/hooded/cultrobes/alt(user), slot_wear_suit)
-	H.equip_or_collect(new /obj/item/weapon/storage/backpack/cultpack(user), slot_back)
+	if(!H.wear_suit) //Only get the equipment if the slot is free.

```

Comment:
These are all kinda unnecessary; `equip_or_collect` ensures that if the slot is filled, it'll try to put it somewhere else.